Source/Fuse.Nodes/Visual.Children.uno

```diff
@@ -258,18 +250,16 @@ namespace Fuse
 					var c = items.Current;
 					if (allowMove)
 					{
-						var where = IndexOf(c);
-						if (where != -1)
+						if (Children_Contains(c))
```

Comment:
(not about this line, about the HashSet)

I wonder if we can't assume that `InsertOrMove` is exclusive and add an intrinsic boolean flag to node to track the `moved` status done here. That would avoid the HashSet.